README.md

```diff
@@ -188,6 +210,31 @@ All configuration values should be specified as strings unless otherwise noted.
         - `"maxFileSizeMB"` An integer type describing how large logs can get
           (in megabytes) before being rotated.
 - `"quickstart"` Configuration options related to the quickstart script
+    - `"Docker"` Configuration options related to Docker
+        - `"enable"` Must be a boolean type. If set to `true`, quickstart
+          will start the servers within a Docker container
+        - `"restart"` Must be a boolean type. If set to `false`, quickstart
+          will not restart the Docker container if it is already started.
+          This is, however, subject to some conditions. If a rebuild is needed
+          (or requested with `-R`), or if the configuration options have
+          changed, the Docker container will be restarted regardless of this
+          option.
```

Comment:
I'm not sure what you mean by this. There is no explicit restart option. There is an explicit rebuild option.